src/main/java/com/google/devtools/build/lib/actions/FileArtifactValue.java

```diff
@@ -177,36 +179,49 @@ public static FileArtifactValue create(Artifact artifact, FileValue fileValue)
       throws IOException {
     boolean isFile = fileValue.isFile();
     FileContentsProxy proxy = getProxyFromFileStateValue(fileValue.realFileStateValue());
-    return create(
+    FileArtifactValue resolvedValue = create(
```

Comment:
Will this change the behavior of symlinks-to-directories? If not, where is `isFile` set to false in that case?